lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -993,6 +995,18 @@ data ApiPostRandomAddressData = ApiPostRandomAddressData
     } deriving (Eq, Generic, Show)
       deriving anyclass NFData
 
+data ApiWalletMigration (n :: NetworkDiscriminant) = ApiWalletMigration
+    { selections :: !(NonEmpty (ApiCoinSelection n))
+    , cost :: !(Quantity "lovelace" Natural)
```

Comment:
> No strong opinion, but if this was named fee it would be consistent with ApiTransaction.

Good point! Perhaps `totalFee` would be even better, because each individual selection has its own fee. What do you think?

Reply:
Changed this to `totalFee` in commit 03696719acfd86b225d6fca8f11766f911a0b1c0.